cg/server/api.py

```diff
@@ -356,6 +366,23 @@ def parse_flow_cell(flowcell_id):
     return jsonify(**flow_cell.to_dict(samples=True))
 
 
+@BLUEPRINT.route("/flowcells/<flow_cell_id>/sequencing_metrics", methods=["GET"])
+def get_sequencing_metrics(flow_cell_id: str):
+    """Return sequencing metrics for a flow cell."""
```

Comment:
```suggestion
def get_sequencing_metrics(flow_cell_id: str):
    """Return sample lane sequencing metrics for a flow cell."""
```
We can also think of making the function name more explicit to what it fetches, i.e. ```get_sample_lane_sequencing_metrics``` 
Although I think ```sequencing_metrics``` can be considered explicit enough with a clarification in the doc string.

Reply:
I think sample_lane_sequencing metrics is a mouthful. For most circumstances i think `sequencing_metrics` is enough

Reply:
Updated the doc string to be more descriptive 👍